common/app/layout/Sublink.scala

```diff
@@ -211,8 +211,8 @@ object FaciaCard {
     showSeriesAndBlogKickers: Boolean
   ): FaciaCard = {
 
-    val isTestContainer = config.displayName.contains("Paid content in unbranded container")
-    if (isTestContainer && faciaContent.branding(defaultEdition).exists(_.isPaid)) {
+    val containerName: Option[String] = config.displayName
+    if ((containerName.contains("Paid content in unbranded container") || containerName.contains("lifestyle")) && faciaContent.branding(defaultEdition).exists(_.isPaid)) {
```

Comment:
Good point, there will be tidy up to do (and more work we think!) after the test finishes so I'll be able to come back to this.